src/mame/machine/archimds.cpp

```diff
@@ -97,30 +97,42 @@ void archimedes_state::device_timer(emu_timer &timer, device_timer_id id, int pa
 {
 	switch (id)
 	{
-		case TIMER_VBLANK: vidc_vblank();break;
-		case TIMER_VIDEO: vidc_video_tick(); break;
-		case TIMER_AUDIO: vidc_audio_tick(); break;
+		//case TIMER_VBLANK: vidc_vblank();break;
+		//case TIMER_VIDEO: vidc_video_tick(); break;
+		//case TIMER_AUDIO: vidc_audio_tick(); break;
```

Comment:
AFAICT these are all triggered by the device now.  Should be safe to remove them, right?